.github/templates/linux_ci_workflow.yml.in

```diff
@@ -0,0 +1,174 @@
+# @generated by .github/scripts/generate_linux_ci_workflows.py, Do not update manually
```

Comment:
This file itself isn't generated, right? The script should add in the bit about how it's a generated file to the `workflow/*.yml` instead of putting it here in the source

Reply:
I'll do this in a follow up PR to this